sql/core/src/test/resources/sql-tests/results/describe.sql.out

```diff
@@ -97,16 +97,21 @@ DESC EXTENDED t
 struct<col_name:string,data_type:string,comment:string>
 -- !query 6 output
 # Detailed Table Information	CatalogTable(
```

Comment:
The output looks weird if we sort the Command, so file a PR: https://github.com/apache/spark/pull/17414